src/dev/yarn_deduplicate/index.ts

```diff
@@ -17,7 +17,7 @@ const yarnLock = readFileSync(yarnLockFile, 'utf-8');
 const output = fixDuplicates(yarnLock, {
   useMostCommon: false,
   excludeScopes: ['@types'],
-  excludePackages: ['axe-core', '@babel/types', 'csstype'],
+  excludePackages: ['axe-core', '@babel/types', 'csstype', 'yaml'],
```

Comment:
without this the deduplicate script was always adding a commit and subsequent bootstrap failed with  error like this 
```
[bazel] ERROR: /.../kibana/packages/kbn-monaco/BUILD.bazel:36:12: Action packages/kbn-monaco/target_workers failed: (Exit 1): webpack-cli.sh failed: error executing command bazel-out/host/bin/external/npm/webpack-cli/bin/webpack-cli.sh --config packages/kbn-monaco/webpack.config.js --output-path bazel-out/darwin_arm64-fastbuild/bin/packages/kbn-monaco/target_workers --env ... (remaining 3 arguments skipped)
[bazel]
[bazel] Use --sandbox_debug to see verbose messages from the sandbox
[bazel]
[bazel] ERROR in ./node_modules/yaml/browser/dist/nodes/Scalar.js 12:19
[bazel] Module parse failed: Unexpected token (12:19)
[bazel] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[bazel] |     }
[bazel] |     toJSON(arg, ctx) {
[bazel] >         return ctx?.keep ? this.value : toJS(this.value, arg, ctx);
[bazel] |     }
[bazel] |     toString() {
[bazel]  @ ./node_modules/yaml/browser/dist/index.js 8:0-43 8:0-43
[bazel]  @ ./node_modules/yaml/browser/index.js
[bazel]  @ ./node_modules/monaco-yaml/yaml.worker.js
[bazel]  @ ./packages/kbn-monaco/src/yaml/worker/yaml.worker.ts
```